src/main/java/nl/tudelft/dnainator/annotation/AnnotationCollection.java

```diff
@@ -0,0 +1,50 @@
+package nl.tudelft.dnainator.annotation;
+
+import java.util.Collection;
+
+/**
+ * An abstract interface for storing annotations.
+ */
+public interface AnnotationCollection {
+
+	/**
+	 * Add annotations from a given source.
+	 * @param source The source of the annotations.
+	 */
+	default void addAnnotations(AnnotationSource source) {
```

Comment:
No, the `GFFLoadService` needs to be able to access an existing `AnnotationCollection` of the `Graph`, and fill it with annotations.